Rubberduck.Parsing/VBA/RubberduckParser.cs

```diff
@@ -127,14 +125,19 @@ public void Parse()
         /// </summary>
         private void ParseAll()
         {
-            var projects = _state.Projects
-                // accessing the code of a protected VBComponent throws a COMException:
-                .Where(project => project.Protection == vbext_ProjectProtection.vbext_pp_none)
-                .ToList();
+            if (!_state.Projects.Any())
+            {
+                foreach (var project in _vbe.VBProjects.UnprotectedProjects())
+                {
+                    _state.AddProject(project);
+                }
+            }
+
+            var projects = _state.Projects.ToList();
 
             var components = projects.SelectMany(p => p.VBComponents.Cast<VBComponent>()).ToList();
-            var modified = components.Where(_state.IsModified).ToList();
-            var unchanged = components.Where(c => !_state.IsModified(c)).ToList();
+            var modified = components.Where(c => _state.IsNewOrModified(c)).ToList();
```

Comment:
perhaps `modified` local variable should be named `newOrModified` for consistency..